client-templates/nexus2/.env.sample

```diff
@@ -0,0 +1,27 @@
+# your unique broker identifier
+BROKER_TOKEN=<broker-token>
+
+# The Base URL for your Nexus Repository Manager
+# If not using basic auth this will only be "https://<your.nexus.hostname>"
+BASE_NEXUS_URL=https://<username>:<password>@<your.nexus.hostname>
+
+# The URL to your Nexus Repository Manager
+NEXUS_URL=$BASE_NEXUS_URL/service/local/repositories
+
+# The URL of the Snyk broker server
+BROKER_SERVER_URL=https://broker.snyk.io
+
+# the fine detail accept rules that allow Snyk to make API requests to your
+# Nexus instance
+ACCEPT=accept.json
+
+# The path for the broker's internal healthcheck URL. Must start with a '/'.
+BROKER_HEALTHCHECK_PATH=/healthcheck
+
+# Provide RES_BODY_URL_SUB with the URL of the Nexus without credentials and http protocol
+# This URL substitution is required for NPM integration
+# RES_BODY_URL_SUB=https://<your.nexus.hostname>/service/local/repositories
+
+BROKER_CLIENT_VALIDATION_URL=$BASE_NEXUS_URL/service/local/status
```

Comment:
need to check this and add some commentry if its different when using auth vs not using auth

Reply:
according to the docs- we can use `'$BASE_NEXUS_URL/internal/healthcheck'`. 
will need to validate if it requires `basicAuth` when QA'ing as well. 
See -  
- https://github.com/sonatype/nexus-oss/blob/nexus-2.11.x/components/nexus-webapp/src/main/java/org/sonatype/nexus/webapp/metrics/MetricsModule.java#L94

- https://blog.sonatype.com/learn-the-nexus-rest-api-automating-sonatype-nexus

Reply:
no need to use the `/internal` endpoint since trying to hit `BROKER_CLIENT_VALIDATION_URL=$BASE_NEXUS_URL/nexus/service/local/status` with basic auth in front of the nexus domain also works, and returning 401 if wrong creds being passed.